src/buffers.h

```diff
@@ -15,7 +15,22 @@
  */
 typedef struct {
     u8 pixel_index_array[0x1000];
-} struct_D_802BFB80; // size = 0x1000
+} struct_D_802BFB80_8; // size = 0x1000
+
+typedef struct {
+    u8 pixel_index_array[0x2000];
+} struct_D_802BFB80_4; // size = 0x1000
+
+/*
+ * This is so fucking dumb I hate that this works
+ * In render_player, spawn_players, and staff_ghosts D_802BFB80 is the arraySize8 entry
+ * But in code_80091750 its the arraySize4 entry, for some unholy reason
+ * The only way to unify those 2 things is to use a union
```

Comment:
Can we modify this comment. I think buffers.h should have clean comments if possible.